src/docfx/build/xref/InternalXrefMapBuilder.cs

```diff
@@ -128,6 +129,19 @@ string GetBookmarkFromUid(string uid)
                 => Regex.Replace(uid, @"\W", "_");
         }
 
+        private static (Error, (bool isRoot, string jsonPath, Dictionary<string, Lazy<JToken>> propertiesByUid)) AggregateXrefSpecProperties(string uid, List<(bool isRoot, string jsonPath, Dictionary<string, Lazy<JToken>> properties)> uidWithProperties)
```

Comment:
can detect `isRoot` by checking `JToken.Parent is null`

Reply:
we don't have the UID associated `JToken` here